README.md

```diff
@@ -65,147 +65,218 @@ TechInfo lookup is looking at Toyota's Techinfo site (payment required, minimum
-      #
-      # if bl_version != APPLICATION_VERSIONS[app_version]:
-      #    print("Unexpected bootloader version!", bl_version)
-      #    exit(1)
-      ```
-    - Save and exit the editor (Ctrl+X, then Y, then Enter).
-    - Run the script again:
-      ```sh
-      ./extract_keys.py
-      ```
-
-10. Manually add the key to params (if needed):
-    - Use the following command to manually change the keys:
-      ```sh
-      echo -n "your key here" > /data/params/d/SecOCKey
-      ```
-
-11. Fingerprinting (if the car is not recognized):
-    - Follow the guide on fingerprinting:
-      https://github.com/commaai/openpilot/wiki/Fingerprinting
-    - Locate the necessary ECU codes.
-    - Add the ECU codes to `fingerprints.py`:
-      ```sh
-      nano /data/openpilot/selfdrive/car/toyota/fingerprints.py
-      ```
-    - Scroll down to the `CAR.TOYOTA_RAV4_PRIME` section and enter your corresponding ECU codes:
-      ```python
-      },
-      CAR.TOYOTA_RAV4_PRIME: {
-        (Ecu.engine, 0x700, None): [
-          b'\x01896634AJ7000\x00\x00\x00\x00',
-          b'\x018966342S7000\x00\x00\x00\x00',
-        ],
-        (Ecu.abs, 0x7b0, None): [
-          b'\x01F15264284100\x00\x00\x00\x00',
-          b'\x01F15264228300\x00\x00\x00\x00',
-        ],
-        (Ecu.eps, 0x7a1, None): [
-          b'\x018965B4233100\x00\x00\x00\x00',
-          b'\x018965B4209000\x00\x00\x00\x00',
-        ],
-        (Ecu.fwdRadar, 0x750, 0xf): [
-          b'\x018821F6201300\x00\x00\x00\x00',
-          b'\x018821F3301400\x00\x00\x00\x00',
-        ],
-        (Ecu.fwdCamera, 0x750, 0x6d): [
-          b'\x028646F4210100\x00\x00\x00\x008646G3305000\x00\x00\x00\x00',
-          b'\x028646F4205200\x00\x00\x00\x008646G4202000\x00\x00\x00\x00',
-        ],
-      ```
-
-12. Disable updates
-
-    ```sh
-    echo -en "1" > /data/params/d/DisableUpdates
-    ```
+1-3. Also connect the OBD2 connector to make sure the Comma stays powered on while turning the car on and off. You can experiment with not using OBD2 once everything's working, but use it for now.
+
+1-4. Turn on your Comma 3X by plugging in the right-angled USB-C cable to the harness.
+
+1-5. Connect it to your Wi-Fi network.
+
+1-6. Install "Custom Software" with URL `commaai/master`
+
+### Step 2. SSH into the device
+2-1. Before you start (for both macOS and Windows): https://github.com/commaai/openpilot/wiki/SSH#before-you-start
+
+2-2-1. macOS: https://github.com/commaai/openpilot/wiki/SSH#option-2mac---pre-installed-openssh-client-on-macos
+
+2-2-2. Windows: https://github.com/commaai/openpilot/wiki/SSH#option-2---pre-installed-openssh-client-on-windows-10-and-up
+
+2-3. SSH into the device.
+```sh
+ssh comma@"your Comma IP"
+```
+
+### Step 3. Extract security key
+> [!TIP]
+> If you already have the security key, proceed to Step 5. 
 
-12. Reboot the device:
-    - Enter the reboot command:
-      ```sh
-      sudo reboot
-      ```
+3-1. Put the car into `Not Ready to Drive` mode.
```

Comment:
Changed from "`Ignition On` mode" because on my '23 Sienna it says "`POWER ON`"

Reply:
ok